boards/arm/stm32h7b3i_dk/Kconfig.defconfig

```diff
@@ -8,10 +8,31 @@ if BOARD_STM32H7B3I_DK
 config BOARD
 	default "stm32h7b3i_dk"
 
+config KSCAN
+	default y if DISPLAY
+
 config I2C
 	default y if KSCAN
 
 config KSCAN_FT5336
 	default y if KSCAN
 
+config KSCAN_FT5336_INTERRUPT
+	default y if KSCAN_FT5336
+
+config STM32_LTDC
+	default y if DISPLAY
+
+config STM32_LTDC_FRAME_BUFFER_IN_EXTERNAL_SDRAM
+	default y if STM32_LTDC
+
+if STM32_LTDC_FRAME_BUFFER_IN_EXTERNAL_SDRAM
+choice STM32_LTDC_FRAME_BUFFER_SDRAM_BANK
+	default STM32_LTDC_FRAME_BUFFER_IN_SDRAM_BANK_2
+endchoice
+endif
+
+config MEMC
+	default y if STM32_LTDC_FRAME_BUFFER_IN_EXTERNAL_SDRAM
```

Comment:
Samples in `samples/drivers/display` and `samples/subsys/display/lvgl` cannot be built without using external SDRAM because frame buffer cannot fit into a single SRAM. 
There are 3 AXI SRAM arrays on STM32H7B3I which are contiguous in terms of address space (their sizes are 256K, 384K and 384K) like shown on the following figure:
![image](https://user-images.githubusercontent.com/33724303/159003865-0587f805-cb3b-4d92-90ab-ee157eab7dd3.png)
Does it make more sense for them to be combined into a single address space in device tree? Because right now only the first SRAM array is used for storing .data and .bss, leaving 768 KB of SRAM unused.

Reply:
One solution would be to overwrite SDRAM definition in a single one in the board dts. Though, I agree this is not quite satisfying.
@carlocaione @JordanYates maybe you guys know a nice way to do this ?

Reply:
uhm, not really. The DTS for the board should declare a new and bigger SDRAM region.

Reply:
Thanks for clarification! Combining contiguous SRAMs into a single address space in an application overlay file seems like a good solution.

Since this board contains an external SDRAM for the purpose of storing the display frame buffer, I'll leave the board DTS like it is.
Custom boards that want to use internal SRAM for storing frame buffer can overwrite SRAM memory regions like you suggested.